Makefile

```diff
@@ -83,6 +100,40 @@ build-cli: ## build PouchContainer cli binary
 	@mkdir -p bin
 	@go build -o bin/${CLI_BINARY_NAME} github.com/alibaba/pouch/cli
 
+dev-image: ## build the Docker Image as cross building environment
+	docker build -f Dockerfile.amd64.cross . -t ${POUCH_IMAGE}
+
+shell: dev-image ## enter into the cross building shell environment
+	docker run -ti --privileged --rm -v ${CURDIR}:/go/src/${POUCH_PRO} \
+		-v ${GOPATH}/src/${RUNC_PRO}:/go/src/${RUNC_PRO} \
+		-v ${GOPATH}/src/${CONTAINERD_PRO}:/go/src/${CONTAINERD_PRO} \
+		${POUCH_IMAGE} \
+		bash
+
+cross-build: dev-image
```

Comment:
need make dev-image every time?

Reply:
Yes, since there is the case that if pouch_dev images existed but without any build dependencies ..... that's sad...